### DISCUSSION TITLE
log4j is supporting the Openjdk21?

### DISCUSSION THREAD

**Discussion by akrattan** [apache/logging-log4j2#2673] [Q&A]
Hello Team,

Which version of log4j is supporting the Openjdk21?

Thanks & Regards

Abhishek

**Comment by ppkarwasz** [comment]
Java versions are mostly backward compatible, so:

- the 2.x branch supports from Java 8 to the latest one (currently 22),
- the 3.x branch supports from Java 17 to the latest one (currently 22).

See also #2121
